### PR TITLE
Update Readme.MD to v1

### PR DESCRIPTION
**Pull request by hakontro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The common options are:
...
 | `limit`       | The number of items to fetch for this resource type to create the DataFrame. Note that this is different from the SQL `SELECT * FROM ... LIMIT 1000` limit. This option specifies the limit for items to fetch from CDF, *before* filtering and other transformations are applied to limit the number of results. |                                            |
 | `batchSize`   | The maximum number of items to read/write per API call.                                                                                                                                                                                                                                                           |                                            |
-| `baseUrl`     | The prefix for all CDF API calls. Default: https://api.cognitedata.com                                                                                                                                                                                                                                            |                                            |
+
```
Comment (wjoel) [inline README.md]: Do we have an issue to add support for this again? The Scala SDK now supports it, so it should be easy.

File: README.md [hunk 2]
```diff
@@ -318,7 +279,7 @@ df.groupBy("fileType").count().show()
 Learn more about 3D models and revisions [here](https://doc.cognitedata.com/dev/concepts/resource_types/3dmodels.html)
 
 Note that the Open Industrial Data project does not have any 3D models. To test this example, you need a project with existing 3D models. There are five options for listing metadata about 3D models:
```
Comment (wjoel) [inline README.md]: "four" now, then. :)

File: README.md (hunk 1 above)
Comment (hakontro) [reply README.md]: We do now! https://github.com/cognitedata/cdp-spark-datasource/issues/315

File: README.md (hunk 2 above)
Comment (hakontro) [reply README.md]: True, fixed!

**Pull request merged**